src/dev/i18n/extractors/code.js

```diff
@@ -61,7 +61,7 @@ export function isFormattedMessageElement(node) {
   return isJSXOpeningElement(node) && isJSXIdentifier(node.name, { name: 'FormattedMessage' });
 }
 
-export function* extractCodeMessages(buffer) {
+export function* extractCodeMessages(buffer, reporter) {
   let ast;
 
   try {
```

Comment:
__suggestion:__ at line 74 (below) we can probably do `reporter.report + return;` instead of throwing "fail error"?